### ISSUE TITLE
[Feature Request] Black Theme / Dark Mode

### ISSUE THREAD

**Issue by jaredi** [microsoft/accessibility-insights-windows#623]
A Black Theme or Dark Mode would help me (and others with light sensitivities) use this tool without inducing migraines. Accessibility Insights is one of the few "bright white" tools I have to use on a regular basis that I cannot theme with a dark background.

I suffer from migraines due to light sensitivity, and a big part of it is applications that only offer a bright white theme.  It’s to the point that I have to often work with sunglasses on, and sometimes even that is not enough.

Many times I can find a replacement, like Notepad++ which offers dark themes.
I do use F.Lux to darken my screen for apps that don't support a Dark Mode, but F.Lux turns everything orange, which isn't ideal when I'm working on visual UI and need to see the true colors of the application I'm actively developing.

Since Accessibility Insights is the best tool for checking UI for accessibility issues, I have to use Accessibility Insights.  There is no replacement.

Several times I have needed to leave early (or take a day off) with a migraine after using bright applications for long periods.  Please consider adding a way to theme the Accessibility Insights application with dark colors.

**Comment by smoralesd** [comment]
Thanks for the feature request @jaredi we'll add this to our feature triage process. In the meantime, have you tried high contrast mode from Windows? I know it's not exactly what you're asking for but it may work as a good workaround for you.

**Comment by jaredi** [comment]
That's a good thought, but unfortunately High Contrast Black Theme is not a good workaround, because it changes everything, including the application I'm currently developing. :)   I need to be able to use the Accessibility Insights tool against my application in both High/Low contrast themes.

**Comment by ferBonnin** [comment]
Hi Jaredi, thanks for the feature requests and the insights on why high contrast is not a good workaround. We are investigating your feature request :)

**Comment by ferBonnin** [comment]
Marking this as ready for work. Design team has provided us with the colors for implementing this feature :)

**Comment by cheade** [comment]
Adding the Figma link to the color palette for [Windows - Dark mode:](https://www.figma.com/file/MxXdGGjWOjE7KaL3rfJNed5C/Accessibility-Insights-for-Windows?node-id=2505%3A0)

**Comment by DaveTryon** [comment]
@jalkire, I've added the framework for selecting the palette at https://github.com/DaveTryon/accessibility-insights-windows/tree/DarkMode, but as you suspected, assigning the palette colors is proving more difficult than I suspected. I won't be looking at this further until I'm back in the office, in case you wanted to see how far you could get with it.

**Comment by DaveTryon** [comment]
Putting this one back into the stack, since I'm actively working on something else.

**Comment by DaveTryon** [comment]
@jaredi @lekevin-microsoft @cwalker-msft @jlyday @aalapc @eylult @martinca-msft @elsypinzonv @wravery, just wanted to let you know that an initial version of dark mode (covering only some of the top level windows) is available for initial feedback. You can get it by switching to the Canary channel and modifying your config file to enable dark mode as described in #719. We'd love to get your feedback on the overall feel. Thanks!

**Comment by jaredi** [comment]
Installed and working.  I'll try it out more today.  It's much more pleasant on my eyes.  Thank you!!

**Comment by ferBonnin** [comment]
hi @jaredi @lekevin-microsoft @cwalker-msft @jlyday @aalapc @eylult @martinca-msft @elsypinzonv @wravery we would love to hear your feedback on the dark mode!
Thanks!

**Comment by cwalker-msft** [comment]
This is a really great start. Please update these boxes to also use dark backgrounds:

![image](https://user-images.githubusercontent.com/57200230/77130154-0ab8e100-6a14-11ea-85aa-8fddc602757e.png)

**Comment by DaveTryon** [comment]
@cwalker-msft, please check out #734

**Comment by ferBonnin** [comment]
received feedback from requester via email

**Comment by DaveTryon** [comment]
#764 has merged into release [1.1.1206.01](https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1206.01), which released to canary this morning. With this change, dark mode is now enabled by default. Closing this issue, but please file separate issues for things that seem wrong in dark mode.
